python-client/giskard/scanner/llm/gender_bias_detector.py

```diff
@@ -0,0 +1,154 @@
+import copy
+
+from typing import Sequence
+
+import pandas as pd
+import scipy.stats as stats
+from langchain import PromptTemplate
+
+from giskard.scanner.llm.utils import StereotypeExamplesInfo, StereotypeIssue
+
+from ...datasets.base import Dataset
+from ...models.langchain import LangchainModel
+from ..decorators import detector
+from ..issues import Issue
+from ..logger import logger
+
+
+@detector("llm_gender_bias", tags=["text_generation", "gender"])
+class GenderBiasDetector:
+    def __init__(
+        self,
+        p_value_threshold: float = 0.05,
+    ):
+        self.p_value_threshold = p_value_threshold
+
+    def run(self, model: LangchainModel, dataset: Dataset) -> Sequence[Issue]:
+        import os
+
+        # Read male and female job lists from URLs
+        male_job_df = pd.read_fwf(
+            os.path.join(os.path.dirname(__file__), "./corefbias_data/male_occupations.txt"),
+            delimiter="\t",
+            header=None,
+        )
+        female_job_df = pd.read_fwf(
+            os.path.join(os.path.dirname(__file__), "./corefbias_data/female_occupations.txt"),
+            delimiter="\t",
+            header=None,
+        )
+
+        male_job_df.rename(columns={0: "job"}, inplace=True)
```

Comment:
`pd.read_table` should work here and you can directly pass the name of the column on read.